ONBOARDING.md

```diff
@@ -1,50 +1,70 @@
 # Node.js build WG onboarding
 
-This document outlines what things future members of the build WG should be
-acquainted with. It might be expanded into additional documentation down
-the road.
+This document is an outline of the things we tell new Build WG members at their
+onboarding session.
 
-### General information
+### Before the onboarding session
 
-  - Responsibilities
-  - Members/"who does what"/interest areas
-  - Technologies and software in use
+* Email the new member and obtain their public GPG key
+* Ask them to start looking over `services.md` and `jenkins-guide.md`,
+  and to try and have any questions ready
+* Add them to the `test` group in `nodejs-private/secrets`
+* Add them to the `nodejs/build` and `nodejs-private/build` GitHub teams
+* Install Ruby and the `dotgpg` gem onto your development machine
```

Comment:
This is something you want them to do and not something the onboarder does (because presumably they already have it set up), right? If so, make that clear here because all the items above are from the perspective of the onboarder and not the onboardee. Same with the next item.